semantic/src/site/globals/site.overrides

```diff
@@ -488,6 +543,8 @@ a:hover {
       Application Summary area
 ---------------------------------*/
 
+// Note: these layout ids/classes also shared with Review page
```

Comment:
I think that's the case only for `summary-section`, `.summary-section-header`, `.summary-page`, `.ui.basic.segment.summary-page-element` and `.summary-section .bound.bottom`. So maybe just move these to the "General section" or even create one specific area for Section/Page elements styles?

Reply:
Ok, just found out that  `application-summary` and  `application-summary-header` are in use on ReviewHeader as well, so probably we can rename to a more general name i.e. `summary-header-container` and `summary-header`

Reply:
And `application-summary-content` on **ReviewPage**, same move to the "General" section, although the name for this one seems concise, right? It is a application-summary-content : )

Reply:
I agree, but again I think it'll be easier to do this all at once at the end.